tests/YotiTestBase.php

```diff
@@ -21,6 +21,7 @@ class YotiTestBase extends WP_UnitTestCase
         'yoti_age_verification' => 0,
         'yoti_company_name' => 'company_name',
         'yoti_pem' => [
+            'name' => 'some-pem-name',
```

Comment:
I'm not sure what this has to do with the PR or commit description and I'm not good enough at PHP to work it out.

Reply:
This is just to allow the form values to pass validation so that I could test the trimming of the text field values - here's what the validation code looks like:

```php
if (empty($pemFile['name']))
{
  $errors['yoti_pem'] = 'PEM file is required.';
}
```

(it seemed like the simplest fix for now)